### PR TITLE
fix(codeowners): Change return type from bytes to string

### PR DESCRIPTION
**Pull request by jangjodi**
Fixes SENTRY-2DGX

### CODE REVIEW

**Review by isabellaenriquez — APPROVED** [review]

File: src/sentry/integrations/github/client.py [hunk 1]
```diff
@@ -632,7 +632,11 @@ def get_file(self, repo: Repository, path: str, ref: str, codeowners: bool = Fal
             headers=headers,
         )
 
-        result = contents.content if codeowners else b64decode(contents["content"]).decode("utf-8")
+        result = (
+            contents.content.decode()
```
Comment (jangjodi) [inline src/sentry/integrations/github/client.py:636]: change content from bytes to string in `get_file` since its return type is supposed to be `str`

**Review by isabellaenriquez — APPROVED** [review]
makes sense to me!

File: src/sentry/integrations/gitlab/client.py [hunk 2]
```diff
@@ -348,7 +348,11 @@ def get_file(self, repo: Repository, path: str, ref: str, codeowners: bool = Fal
         )
         contents = self.get(request_path, params={"ref": ref}, raw_response=True, headers=headers)
 
-        result = contents.content if codeowners else b64decode(contents["content"]).decode("utf-8")
+        result = (
+            contents.content.decode()
```
Comment (wedamija) [inline src/sentry/integrations/gitlab/client.py:352]: Should this be `decode("utf-8")`?

Comment (jangjodi) [reply src/sentry/integrations/gitlab/client.py:352]: I think this works either way, but I can add it in another PR

Comment (jangjodi) [reply src/sentry/integrations/gitlab/client.py:352]: here's the follow up PR: https://github.com/getsentry/sentry/pull/62986

**Pull request merged**